tensorflow_datasets/video/bair_robot_pushing.py

```diff
@@ -89,17 +90,17 @@ def _split_generators(self, dl_manager):
 
   def _generate_examples(self, filedir):
     logging.info("Reading data from %s.", filedir)
-    files = tf.io.gfile.listdir(filedir)
+    files = compat.v2.io.gfile.listdir(filedir)
     logging.info("%d files found.", len(files))
 
     # For each file
-    for filename in sorted(tf.io.gfile.listdir(filedir)):
+    for filename in sorted(compat.v2.io.gfile.listdir(filedir)):
       filepath = os.path.join(filedir, filename)
 
       # For each video inside the file
       for video_id, example_str in enumerate(
-          tf.compat.v1.io.tf_record_iterator(filepath)):
-        example = tf.train.SequenceExample.FromString(example_str)
```

Comment:
Here it was correctly pointing to `tensorflow.compat.v2.train`.

Reply:
But again [here](https://github.com/tensorflow/datasets/blob/master/tensorflow_datasets/video/bair_robot_pushing.py#L60-L61) `tf.float32` points to `tensorflow.compat.v2.float32`

Reply:
Could you share the stacktrace ? I think your TF installation might be broken.

Reply:
Actually I was running it on google cloud. But it didn't allow me to use TF > 2.0.0b1 there. (maybe because I was using python3.5 ?)
Allow me some time I will try doing it again with the latest TF and report back to you!

Reply:
Using a `beta` version is likely the issue. Note that TFDS do not support Python 3.5 anymore.

Reply:
locally with `TF 2.2.0`. I was able to access `tf.compat.v2.compat.v1` and `tf.compat.v1.float32`. So it looks like a problem with older versions.
Closing the PR.